solidity/test/bridge/Bridge.MovingFunds.test.ts

```diff
@@ -3067,7 +3067,13 @@ describe("Bridge - Moving funds", () => {
 
                           // Set the max fee to one satoshi less than the fee
                           // used by the transaction.
-                          await bridge.setMovedFundsSweepTxMaxTotalFee(1999)
+                          await bridgeGovernance
+                            .connect(governance)
+                            .beginMovedFundsSweepTxMaxTotalFeeUpdate(1999)
+                          await increaseTime(constants.governanceDelay)
+                          await bridgeGovernance
+                            .connect(governance)
+                            .finalizeMovedFundsSweepTxMaxTotalFeeUpdate()
```

Comment:
With the `BridgeStub` contract size ignored, shouldn't we restore `setMovedFundsSweepTxMaxTotalFee` in the `BridgeStub`? I am fine either way but would be good to have consistency and do not add `setDepositRevealAheadPeriod` if we decide to drop `setMovedFundsSweepTxMaxTotalFee` and always go through the governance contract.

Reply:
Reverted 20df3905